[26기 장연정] westagram login & main 페이지 셀프 리팩토링 완료

yeonjeongj opened the pull request:
>wecode PR message template 및 체크 리스트 입니다. 
아래 사항들을 전부 작성/체크 하시고 PR 해주세요!

## 수정 사항 간략한 한줄 요약
- 로그인 & 메인 페이지 셀프 리팩토링 완료
- 메인 페이지 댓글 작성 및 삭제 부분 이벤트 구현 완
		 
## 수정 사항들 자세한 내용
- 로그인 페이지 html & css & js 셀프 리팩토링 완료
- 로그인 페이지 id & pw 모두 입력 후 id 부분 input 삭제 시 버튼 비활성화가 되지 않는 오류 수정
- 메인 페이지 html & css 셀프 리팩토링 완료
- 메인 페이지 게시 버튼 클릭 혹은 엔터키 누를 시 댓글이 입력되는 이벤트 구현
- 메인 페이지 새로운 댓글 우측 x 버튼 누르면 댓글 삭제되는 이벤트 구현

## 기타 질문 및 특이 사항
- 웹페이지 사이즈 조정 시, 구현한 메인 페이지의 디자인이 흐트러지는 문제가 발생 (반응형 웹 구현 필요)

## 체크 리스트 (아래 사항들이 전부 체크되어야만 merge가 됩니다!)
- [x] 필요한 test들을 완료하였고 기능이 제대로 실행되는지 확인 하였습니다.
- [x] Wecode의 코드 스타일 가이드에 맞추어 코드를 작성 하였습니다.
- [x] 제가 의도한 파일들과 수정 사항들만 커밋이 된 것을 확인 하였습니다.
- [ ] 본 수정 사항들을 팀원들과 사전에 상의하였고 팀원들 모두 해당 PR에 대하여 알고 있습니다.

joonsikyang commented:
PR 제목 수정해주시고 라벨도 바꿔주세요!

dipsiiiiiiiiii commented on students/13th/suhyun/js/login.js:
```diff
@@ -0,0 +1,39 @@
+const thisIsID = document.getElementById('ID')
+const thisIsPW = document.getElementById('PW')
+const thisIsButton = document.getElementById('btn')
```
변수 이름에 "이거는 id이다"라고 해줄 필요는 없는 것 같습니다!
예를 들어서 ```idInput``` 이면 한 번에 알 수 있겠죠?

dipsiiiiiiiiii commented on students/13th/suhyun/js/login.js:
```diff
@@ -0,0 +1,39 @@
+const thisIsButton = document.getElementById('btn')
+
+thisIsPW.addEventListener('input', function() {
+    const idBoolean = document.getElementById('ID').value;
+    const pwBoolean = document.getElementById('PW').value;
```
id,pw input은 계속 사용하니까 위에서 한 번만 가져오시면 될 것 같아요!

dipsiiiiiiiiii commented on students/13th/suhyun/js/login.js:
```diff
@@ -0,0 +1,39 @@
+thisIsID.addEventListener('input', function() {
+    const idBoolean = document.getElementById('ID').value;
+    const pwBoolean = document.getElementById('PW').value;
+    if  (!!idBoolean === true && !!pwBoolean === true) {
+        btn.style.backgroundColor = 'Blue'
+        btn.style.color = 'white'
+    }
```
계속 사용하는 함수라서 따로 빼서 정의하고 사용하시면 좋을 것 같아요!

dipsiiiiiiiiii commented on students/13th/suhyun/login.html:
```diff
@@ -0,0 +1,30 @@
+    <h1 class="logo">
+      <img class="logo_img" src="img/logo_text.png">
+    </h1>
+    <div class="loginInput">
+      <input id="ID" type="text" placeholder=" 전화번호, 사용자 이름 또는 이메일">
+    </div>
+    <div class="loginInput">
+      <input id="PW" type="password" placeholder=" 비밀번호">
+    </div>
+    <div class="loginButton">
+      <button id="btn">로그인</button>
+    </div>
+    <div class="loginA">
+      <a href="forgot.html">비밀번호를 잊으셨나요?</a>
+    </div>
```
div로 다 감싸주셨는데 다 감쌀 필요가 있을까요!?

dipsiiiiiiiiii commented on students/13th/suhyun/main.html:
```diff
@@ -0,0 +1,51 @@
+    />
+  </head>
+  <body>
+    <nav>
```
nav 좋아요~!

dipsiiiiiiiiii commented on students/13th/suhyun/main.html:
```diff
@@ -0,0 +1,51 @@
+        </span>
+      </div>
+    </nav>
+    <main>
```
main 태그 대박!

dipsiiiiiiiiii reviewed (commented):
리뷰 반영해주세요 :)
고생하셨습니다!

Commit:
dddd

Commit:
Add: 로그인 페이지 레이아웃 구현
- 로그인 페이지 전체 레이아웃 구현
- 로그인 페이지 전체 디자인 구현

Add: 로그인 페이지 텍스트 입력 시 버튼 활성화 여부 기능 추가
- ID, PW 텍스트 입력 시 버튼  활성화, 텍스트 삭제 시 버튼  비활성화 되는 이벤트 구현

Add: 메인 페이지 레이아웃 구현
- 메인 페이지 전체 레이아웃 구현
- 메인 페이지 전체 디자인 구현

Commit:
Refactor : login & main  페이지 html & css 코드 리팩토링
- <div> <span> 태그 중 일부 semantic tag로 변경
- 누락된 self-closing tag 추가
- 가독성을 위해 일부 변수명 수정
- 변수명 변경에 따라 발생한 css 코드 수정사항 반영
- 레이아웃 구성요소 bottom-up 방식으로 css 수정
- css 속성 중요도에 따라 순서 변경
- 불필요 코드 제거

Refactor : login 페이지 JS 코드 리팩토링
- 가독성을 위해 일부 함수명 수정
- 코드 전체 구조 리팩토링

Fix : login 페이지 JS 코드 오류 해결
- 버튼이 활성화된 상태에서 ID 부분을 지우더라도 버튼 비활성화가 되지 않는 오류 수정

Modify : login & main 페이지 내 script 태그 위치 수정
- head 태그 내에 있던 script 태그를 body 태그 내 가장 하위로 이동

Add : main 페이지 댓글 추가 및 삭제 이벤트 구현
- 게시 버튼 클릭 혹은 엔터를 누를 경우, 피드 하단에 댓글이 업로드 되는 이벤트 구현
- 업로드 된 댓글을 삭제하는 이벤트 구현
